backends/audiocd/sdl/sdl-audiocd.cpp

```diff
@@ -123,6 +123,12 @@ bool SdlAudioCDManager::play(int track, int numLoops, int startFrame, int durati
 	return true;
 }
 
+bool SdlAudioCDManager::playAbsolute(int startFrame, int numLoops, int duration, bool onlyEmulate,
+		Audio::Mixer::SoundType soundType) {
+	// TODO not implemented
```

Comment:
I am confused with these. To me, it seems that they will override the default method, e.g. emulation, and will be no-op. Am I reading the code correctly?

Reply:
I'd gotten confused by the inheritance in a previous iteration and didn't realize I could safely omit `playAbsolute` implementations. I've deleted these now. 

I checked the SDL docs and see how I think I could implement this method for real, but I also don't have access to a Linux machine to test.